utils/src/unit-test/java/uk/gov/moj/sdt/utils/AbstractSdtUnitTestBase.java

```diff
@@ -134,12 +106,22 @@ protected String dos2Unix(final String in) {
      */
     public Object getAccessibleField(final Class<?> clazzUnderTest, final String fieldName,
                                     final Class<?> clazzOfField, final Object target) {
+        Object object = null;
+
         Field field = ReflectionUtils.findField(clazzUnderTest, fieldName);
-        if (null == field) {
-            field = ReflectionUtils.findField(clazzUnderTest, fieldName, clazzOfField);
+        try {
+            if (null == field) {
+                field = ReflectionUtils.findField(clazzUnderTest, fieldName, clazzOfField);
+            }
+            ReflectionUtils.makeAccessible(field);
+            object = ReflectionUtils.getField(field, target);
+            if (null == object) {
+                throw new RuntimeException("Field " + fieldName + " does not exist");
+            }
+        } catch (Exception e) {
+            throw new RuntimeException("Field " + fieldName + " does not exist or is not accessible");
```

Comment:
What's the purpose of the try-catch block?  Looks a bit odd as it immediately re-throws the RuntimeException raised inside it with a slightly different error message.  Is it trying to catch exceptions thrown by ReflectionUtils?